src/lxc/bdev/lxcoverlay.c

```diff
@@ -133,14 +135,15 @@ int ovl_clonepaths(struct bdev *orig, struct bdev *new, const char *oldname,
 			WARN("Failed to update ownership of %s", work);
 		free(work);
 
+                src = lxc_storage_get_path(orig->src, orig->type);
 		// the src will be 'overlayfs:lowerdir:upperdir'
```

Comment:
Yeah, `/**` is the preferred comment format. However, we only change such things in non-functional commits and only when we have reason to touch that codebase. I need to change some more stuff in this part of the code at some point.